### PR TITLE
Adds color change to table

### LINKED ISSUE #3: Add color based on if change is negative or positive
Green if positive change, red if negative change

### PR DESCRIPTION
**Pull request by amhed**
Fixes #3 and #7

### CODE REVIEW

File: libs/helpers.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 const opt = require('./../libs/models/options.js')
 
 function numberWithCommas(x) {
-    var parts = x.toString().split(".");
-    parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-    return parts.join(".");
-  }
+  const parts = x.toString().split('.');
+  parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return parts.join('.');
```
Comment (amhed) [inline libs/helpers.js]: Didn't know which convention you preferred, single or double. Went with the most used one (singles)

Comment (hminaya) [reply libs/helpers.js]: I'm fine with both

Comment (hminaya) [reply libs/helpers.js]: Please remove the yarn.lock file for now

Comment (hminaya) [reply libs/helpers.js]: I'll take care of it, don't worry

**Pull request merged**